src/tasks.js

```diff
@@ -89,10 +87,14 @@ ko.tasks = (function () {
             }
         },
 
-        // For testing only: return the queue length
-        length: function () {
-            return taskQueueLength;
-        }
+        // For testing only: reset the queue and return the previous queue length
+        reset: function () {
```

Comment:
Is 'reset' exported? Doesn't appear to be exported explicitly, but the fact that specs use it suggests its name doesn't get minified. If this is really only for testing, could we rename it to something like 'resetForTesting'? Otherwise, with such a important-sounding name, people will probably start using it and relying on implementation quirks.